client/src/store/reducers/loginModal.js

```diff
@@ -0,0 +1,15 @@
+import { LOGIN_MODAL } from '../actionTypes';
+
+export default function loginModalReducer(
+	state = {
+		isOpen: false,
+	},
+	action,
+) {
+	switch (action.type) {
+		case LOGIN_MODAL.SET_IS_OPEN:
+			return { ...state, isOpen: action.payload.isOpen };
```

Comment:
From [loginModalAction.js](https://github.com/CasperDash/casperdash-client/pull/153/files#diff-de96ad44a55ca0cf82f5d5cd8bc6cad8422f54a1462523d328c3286207c19c5b), this can be:
```
return { ...state, isOpen: action.payload }
```